### ISSUE TITLE
Ignore tracing for certain Urls /hostname

### ISSUE THREAD

**Issue by sudhirpandey** [awslabs/aws-lambda-powertools-roadmap#35]
**Is your feature request related to a problem? Please describe.**
We are using a capture lambda handler decorations, but unfortunately our lambda is making a large amount of api calls to ec2.amazon.com, which we dont want any trace(segements) for.  Since a large amount of  trace subsegment is generate due to this api call( that happens in loop), we run out of xray limit for  a signel trace which is 500Kb .
https://docs.aws.amazon.com/general/latest/gr/xray.html#limits_xray.

**Describe the solution you'd like**
If we could avoid generating trace segements for a known URL /hostname as described here https://github.com/aws/aws-xray-sdk-python#ignoring-httplib-requests, while using tracer , we would be able to get our trace limit under the limit and still use tracer. Its possible for us to use xray sdk directly and start tracing only methods we wanted but, it would be very useful to have such ignore capability in tracer too.

**Comment by michaelbrewer** [comment]
@heitorlessa - would the UX be something like this?

```python3
from aws_lambda_powertools import Tracer

tracer = Tracer()
# Uses `from aws_xray_sdk.ext.httplib import add_ignored as xray_add_ignored`
tracer.add_ignored(hostname="ec2.amazon.com")  # or add_httplib_ignored ?

def ec2_api_calls():
    return "suppress_api_responses"

@tracer.capture_lambda_handler
def handler(event, context):
    for x in long_list:
        ec2_api_calls()
```

alternatively your suggestion is also useful to suppress the responses:

```python3
from aws_lambda_powertools import Tracer

tracer = Tracer()

@tracer.capture_method(capture_response=False)
def ec2_api_calls():
    return "suppress_api_responses"

@tracer.capture_lambda_handler(capture_response=False)
def handler(event, context):
    for x in long_list:
        ec2_api_calls()

```

**Comment by heitorlessa** [comment]
I think it needs something more explicit for naming, like 

tracer.ignore_endpoints(hostnames=[...])

But to make it future proof, it'd be interesting to see how and whether OpenTelemetry has this capability too.

**Comment by michaelbrewer** [comment]
@heitorlessa @sudhirpandey - a PR is up for this https://github.com/awslabs/aws-lambda-powertools-python/pull/910

**Comment by heitorlessa** [comment]
this is now merged - it'll be available in the next release!

**Comment by heitorlessa** [comment]
hey @sudhirpandey thanks a lot for the feature request, @michaelbrewer for the implementation - this is available in the 1.24 release
